.xo-config.json

```diff
@@ -0,0 +1,47 @@
+{
+  "prettier": true,
+  "space": 2,
+  "plugins": ["import"],
+  "rules": {
+    "n/no-unsupported-features": "off",
+    "n/no-unsupported-features/node-builtins": "off",
+    "n/file-extension-in-import": "off",
+    "sort-imports": [
+      "error",
+      {
+        "ignoreCase": false,
+        "ignoreDeclarationSort": true,
+        "ignoreMemberSort": false,
+        "memberSyntaxSortOrder": ["none", "all", "multiple", "single"],
+        "allowSeparatedGroups": false
+      }
+    ],
+    "import/no-named-as-default": "off",
+    "import/extensions": "off",
+    "import/order": [
+      "error",
+      {
+        "groups": ["builtin", "external", "parent", "sibling", "index"],
+        "alphabetize": {
+          "order": "asc",
+          "caseInsensitive": true
+        },
+        "warnOnUnassignedImports": true,
+        "newlines-between": "never"
+      }
+    ]
+  },
+  "overrides": [
+    {
+      "files": ["sdk.mjs"],
+      "nodeVersion": ">=14"
+    },
+    {
+      "files": ["scripts/**/*", "tests/**/*"],
+      "nodeVersion": ">=18",
+      "rules": {
+        "unicorn/no-process-exit": "off"
```

Comment:
Why do you've ignored this rule. It seems to me that is fine to add, just we need to add `#!/usr/bin/env node` to the scripts, see https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-process-exit.md

Reply:
Then you should change those files to executable files. I don't think it's reasonable to have a shebang for a file that can't be executed.

Reply:
What do you mean with "for a file that can't be executed". Yes, can be executed using `./scripts/file.js` directly if we add shebangs for them.

Reply:
> and make them executables

I think this is what @LitoMore was getting at, we would have to make them executable with `chmod +x`. Simple fix though and I think it makes sense!

Reply:
If you agree to update those `node scripts/file.js` to `./scripts/files.js` with `chmod +x`, then it's OK.

Reply:
Yes, please do it 👍🏼 just `+x` is enough!